lib/utilities/ts/calculate-value.ts

```diff
@@ -0,0 +1,13 @@
+const calculateValue = (value: string, maxLength: number): string => {
```

Comment:
calculateValue is looking now a bit vague name. What it really does is truncating a string, isnt it?

Reply:
Gotcha, I rename this 'truncateString' as per your suggestion